src/ts/view.ts

```diff
@@ -132,6 +132,18 @@ document.addEventListener('DOMContentLoaded', () => {
   // sort tables in entity bodies
   // primarily from https://stackoverflow.com/a/49041392
   const getCellValue = (tr, idx) => {
+    // handle colspans
+    const cells = tr.querySelectorAll(':scope > th, :scope > td');
+    let idxMax = 0;
+    let idxMin = 0;
+    for (const cell of cells) {
+      idxMax += cell.colSpan;
+      idxMin = idxMax - cell.colSpan;
```

Comment:
change order to avoid + and - cell.colSpan:
~~~javascript
idxMin = idxMax;
idxMax += cell.colSpan;
~~~